packages/freezed/example/lib/main.dart

```diff
@@ -89,8 +99,72 @@ void main() {
   // nice toString
   print(const Union(42)); // Union(value: 42)
   print(const Union.loading()); // Union.loading()
-  print(const Union.error(
-      'Failed to fetch')); // Union.error(message: Failed to fetch)
+  print(const Union.error('Failed to fetch')); // Union.error(message: Failed to fetch)
+
+  // union with single case - with data
+  const singleCaseUnionWithDataExample = SingleCaseUnionWithData.singleCase(43, 'string');
```

Comment:
Do we really need to add this to `freezed/example/lib/main.dart`? It sounds redundant with the previous part.